src/libs/ReportUtils.ts

```diff
@@ -2906,7 +2907,7 @@ function canEditMoneyRequest(reportAction: OnyxInputOrEntry<ReportAction<typeof
         return true;
     }
 
-    if (policy?.type === CONST.POLICY.TYPE.CORPORATE && moneyRequestReport && isCurrentUserSubmitter(moneyRequestReport.reportID)) {
+    if (policy?.type === CONST.POLICY.TYPE.CORPORATE && moneyRequestReport && isSubmitted && isCurrentUserSubmitter(moneyRequestReport.reportID)) {
         const isForwarded = PolicyUtils.getSubmitToAccountID(policy, moneyRequestReport.ownerAccountID ?? 0) !== moneyRequestReport.managerID;
```

Comment:
NAB since it's an edge case, but to match the [updated backend](https://github.com/Expensify/Auth/blob/820bb60ff3dbf18efb5ca03a0590d78ec23cd974/auth/lib/Report.cpp#L2664), we should return a value which is `!PolicyUtils.hasStrictAdvancedApproval || !isForwarded`.

Otherwise there could be a problem on a corporate policy, delayed submission, if the submitter breaks the workflow and submits to someone else.

Reply:
Agreed, but looks like we'll need a backend PR to include `strictWorkflow` [here](https://github.com/Expensify/Web-Expensify/blob/cba338fddc44c875f2c3dd34c663627975564cef/lib/AppInit.php#L85). How about we merge this and I follow up with a 2nd PR as well as a the Web-E PR?

Reply:
https://github.com/Expensify/App/issues/47517